neo/test/coretest/test_container.py

```diff
@@ -4,8 +4,12 @@
 
 import unittest
 
+import quantities as q
```

Comment:
Conventionally this should be pq
```suggestion
import quantities as pq
```